tests/test_language_id.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from floss.language_identifier import is_go_bin
+
+
+@pytest.mark.parametrize(
+    "binary_file, expected_result",
+    [
+        ("./tests/data/src/go-hello/bin/go-hello.exe", True),
+        ("./tests/data/src/go-hello/bin/go-hello", False),
+        ("test-decode-to-stack.exe", False),
```

Comment:
shouldn't this be also in the same directory as the other binaries? do we need to assert, paths exist?

Reply:
I added the file check. in test script and in try/except